ldap/servers/slapd/slapi-plugin-compat4.h

```diff
@@ -31,16 +31,6 @@ extern "C" {
 #define SLAPI_BIND_FAIL_OR_ANONYMOUS 1 /* back end should send result */
 
 
-/******* Deprecated (obsolete) pblock arg identifiers ************************/
-#define SLAPI_CONFIG_FILENAME 40 /* use config. entries instead */
-#define SLAPI_CONFIG_LINENO 41   /* use config. entries instead */
-#define SLAPI_CONFIG_ARGC 42     /* use config. entries instead */
-#define SLAPI_CONFIG_ARGV 43     /* use config. entries instead */
-#define SLAPI_CONN_AUTHTYPE 144  /* use SLAPI_CONN_AUTHTYPE */
-#define SLAPI_REQUESTOR_ISUPDATEDN SLAPI_IS_REPLICATED_OPERATION
-#define SLAPI_CONN_CLIENTIP 145 /* use SLAPI_CONN_CLIENTNETADDR */
-#define SLAPI_CONN_SERVERIP 146 /* use SLAPI_CONN_SERVERNETADDR */
-
```

Comment:
I think we should mention the removal in the commit message.

But also, we have some leftovers in `test/libslapd/pblock/pblock_accessors*` and a bit in `_slapi_pblock_deprecated` in `ldap/servers/slapd/pblock_v3.h`.

IMO, it'll be cleaner to remove the code in a separate PR.

Reply:
I left this as is with the new commit